persist-cli-tests/src/test/resources/test-src/output/tool_test_migrate_3/persist/migrations/20240401060627_secondMigration/script.sql

```diff
@@ -0,0 +1,13 @@
+-- AUTO-GENERATED FILE.
+-- This file is an auto-generated file by Ballerina persistence layer for the migrate command.
+-- Please verify the generated scripts and execute them against the target DB server.
+
+
+CREATE TABLE `Car` (
+	`id` INT NOT NULL,
+	`make` VARCHAR(191) NOT NULL,
+	`model` VARCHAR(191) NOT NULL,
+	`year` INT NOT NULL,
+	PRIMARY KEY(`id`)
```

Comment:
```suggestion
    `id` INT NOT NULL,
    `make` VARCHAR(191) NOT NULL,
    `model` VARCHAR(191) NOT NULL,
    `year` INT NOT NULL,
    PRIMARY KEY(`id`)
```

Reply:
shall we check other places

Reply:
As discussed, this is because we added a tab for indentation. Agreed to keep as tab